BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -306,34 +305,17 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
         }
     }
     
+    private let lastUpdatedDF: RelativeDateTimeFormatter = {
```

Comment:
```suggestion
    private let lastUpdatedDateFormatter: RelativeDateTimeFormatter = {
```

It took me a minute to work out what DF meant here. In a world of autocomplete, extra characters are cheap, so let's bias against abbreviations/acroyms unless they as really common.